pkg/util/wimi/iswimi_test.go

```diff
@@ -0,0 +1,49 @@
+package wimi
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func TestIswimi(t *testing.T) {
```

Comment:
Suuuper NIT:- Since the function is named `isWimi`, this can be changed to `TestIsWimi`.
Please feel free to dismiss it or handle it accordingly. ```suggestion
func TestIsWimi(t *testing.T) {
```